modules/ort/src/main/kotlin/workflow/processors/enricher/OrtDownloaderProcessor.kt

```diff
@@ -61,6 +61,13 @@ class OrtDownloaderProcessor : AbstractProcessor() {
             try {
                 LOGGER.debug("Download sources via ORT Downloader for '${name}'")
                 val ortDownloadDirectory = createTempDir("ortDownloaderDirectory").also { it.deleteOnExit() }
+                // In some cases, the protocol of a VcsInfo.url is SSH, which leads to hanging builds,
+                // because it's prompting for a password. To avoid this, we are replacing the protocol from ssh
+                // to https for github.com hosts.
+                if (pkg.vcsProcessed.url.startsWith("ssh://git@github.com")){
+                    val newUrl = pkg.vcsProcessed.url.replace("ssh://git@github.com", "https://git@github.com")
```

Comment:
We need to drop the generic `git` username from the `https` URL as

1. in contrast to `ssh`, `https` uses the GitHub username for authentication, and
2. we do not want / need to authenticate at all, as we only clone, but not push.